src/stores/stateChannels.js

```diff
@@ -31,7 +31,14 @@ export const useStateChannelsStore = defineStore('stateChannels', () => {
     rawStateChannels.value = data
   }
 
+  async function fetchStateChannelsCount() {
+    const { data } = await axios.get(`${useRuntimeConfig().public.MIDDLEWARE_URL}/v2/txs/count?tx_type=channel_create`)
```

Comment:
how about we move useRuntimeConfig().public to the top of the store definition and leave here only the MIDDLEWARE_URL?

Reply:
Right, it was already pre-defined in file. Fixed